mkdocs.yml

```diff
@@ -74,5 +74,6 @@ pages:
         - 'wifi': 'en/modules/wifi.md'
         - 'ws2801': 'en/modules/ws2801.md'
         - 'ws2812': 'en/modules/ws2812.md'
+        - 'tm1829': 'en/modules/tm1829.md'
```

Comment:
Sorry for nagging but you should really study the contributing guidelines I pointed you to. The module list here is alphabetically sorted as well.